sidebars.js

```diff
@@ -9,37 +9,6 @@ module.exports = {
       link: {type:"doc", id:"whats-new/release-notes/release-notes-overview"},
```

Comment:
I don't think removing the v2.x release notes in v3 is user friendly. Many users would want to see release notes for v2.x versions

Reply:
Kept the V2 release notes for now.